src/TouristInfoCards.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+const TouristInfoCards = () => {
```

Comment:
Rather than hard coding each card individually you could have stored the data in an Array of Objects and using a map function. This would save you lines of code and would make it easy to add additional cards in the future